lib/src/main/java/com/auth0/jwt/JWTVerifier.java

```diff
@@ -316,17 +325,20 @@ private void assertNonNull(String name) {
             }
         }
 
-        private void addLeewayToDateClaims() {
-            if (!claims.containsKey(PublicClaims.EXPIRES_AT)) {
-                claims.put(PublicClaims.EXPIRES_AT, defaultLeeway);
-            }
-            if (!claims.containsKey(PublicClaims.NOT_BEFORE)) {
-                claims.put(PublicClaims.NOT_BEFORE, defaultLeeway);
-            }
-            if (!claims.containsKey(PublicClaims.ISSUED_AT)) {
-                claims.put(PublicClaims.ISSUED_AT, defaultLeeway);
-            }
-        }
+		private void addLeewayToDateClaims() {
```

Comment:
this method is still not formatted according to the rest of the file. Please run the code formatter or do it manually. Thanks